recipes/bacpage/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set version = "2023.11.10.1" %}
+
+package:
+  name: bacpage
+  version: {{ version }}
+
+source:
+  url: https://github.com/CholGen/bacpage/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 7a62278131407195a8f7555a92ed2d86ca7e63366bac6d280dac337e5f5218c5 
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install --no-deps --ignore-installed ."
+  run_exports:
+    - {{ pin_subpackage('bacpage') }}
+
+requirements:
+  host:
+    - python <3.12,>=3.9
+    - pip
+  run:
+    - python <3.12,>=3.9
+    - bc
+    - bcftools =1.17
+    - bedtools =2.25.0
+    - biopython
+    - bwa =0.7.17
+    - emboss =6.6.0
+    - fastqc
+    - iqtree =2.2.3
+    - multiqc
+    - pandas
+    - qualimap
+    - samtools =1.17
+    - snakemake-minimal =7.31.0
+    - snp-sites
+    - fastp
+    - lighter
+    - flash
+    - unicycler
+    - prokka
+    - quast
+    - abricate
+
+test:
+  commands:
+    - bacpage -h
+    - bacpage version
+    - bacpage example "$(mktemp -d)"
+
+about:
+  home: https://github.com/CholGen/bacpage
+  license: GNU-GPL-3
```

Comment:
Can you please use SPDX identifiers: https://spdx.org/licenses/GPL-3.0-or-later.html

Reply:
Ah, will do!